modules/installation-restricted-network-samples.adoc

```diff
@@ -41,7 +41,13 @@ $ oc create configmap registry-config --from-file=$path/ca.crt -n openshift-conf
 $ oc patch image.config.openshift.io/cluster --patch '{"spec":{"additionalTrustedCA":{"name":"registry-config"}}}' --type=merge
 ----
 +
-//(meaning https://github.com/openshift/api/blob/master/config/v1/types_image.go#L46 … add ref to associated doc)
+// PLACEHOLDER
+* Access a list of available images:
++
+----
+oc get is -n openshift -o yaml | grep registry.redhat.io | grep name | | awk '{ print $2 }'
```

Comment:
Duplicate pipeline strings.

Reply:
Yeah I think we need to remove this place holder.

I still need to circle back with some folks if we should bother providing the list (though I think the answer will be yes there) and then what is the best way to go about it.

Reply:
@bparees per our recent email thread ... should we attempt to provide a non-fidly scripting solution here in the doc, or punt on that as well ?  I'm leaning on punting, but did not want to assume

Reply:
yeah i'd say providing them the command to _get_ the list of pullspecs, and pointing them towards to mirror command, is sufficient.  We don't need to provide a full script that's going to be a maintenance headache, I don't think.

Reply:
OK ... let's use this as the command @bmcelvee as an example to access a specific imagestream:

`oc get is ruby -n openshift -o json | jq .spec.tags[].from.name | grep  registry.redhat.io`

Note, I did confirm that by default all the imagestreams either point to images from the payload or images from registry.redhat.io

I was able to do that from via `oc get is -n openshift -o name | xargs -l -r oc get -n openshift -o yaml | grep dockerImageReference`

@bmcelvee